### DISCUSSION TITLE
Missing elevation data?

### DISCUSSION THREAD

**Discussion by eisams** [organicmaps/organicmaps#801] [Q&A] [answered]
Does Organic Maps take into elevation data?

I was visiting a cabin where you have to walk a trail that goes up by 50-60 meters. Organic Maps showed it as 0 m elevation change. Interestingly Google Maps does the same for walking instructions, but if you switch to cycling directions Google shows the real incline change

Location Norway

**Comment by 1papaya** [comment]
Could you be more specific with an example start/end location?

**Comment by eisams** [comment]
Sure,
Try
from: 60.125535, 10.407466
To: 60.130076, 10.412299

Interestingly if I try from my current location I get this elevation map

Maybe it is cut off above 60 degree north?

![B6125111-5BC2-4ECC-B626-04EFBEA7719D](https://user-images.githubusercontent.com/62468530/124396586-bcb01580-dd0a-11eb-9979-642beaf63eeb.png)

**Comment by eisams** [comment, marked as answer]
This will be fixed in the next data update as per issue #830
